typings.json

```diff
@@ -2,8 +2,6 @@
   "globalDependencies": {
     "clean-css": "registry:dt/clean-css#3.4.9+20160316155526",
     "clone": "registry:dt/clone#0.1.11+20160317120654",
-    "express": "registry:dt/express#4.0.0+20160317120654",
-    "express-serve-static-core": "registry:dt/express-serve-static-core#0.0.0+20160322035842",
     "glob": "registry:dt/glob#5.0.10+20160317120654",
```

Comment:
I know a lot of these exist within the npm `@types` ecosystem, mind removing as many as you can?